resources/views/components/matrix-choice.blade.php

```diff
@@ -18,26 +18,27 @@
             </tr>
             </thead>
             <tbody>
-                @foreach($rowData as $rowKey => $rowValue)
-                    <tr>
-                        <td class="p-2">{{ $rowValue }}</td>
-                        @foreach($columnData as $columnKey => $columnValue)
-                            @php
-                                $supStatPath = ($pilColor === 'radio') ? $statePath.'.'.$rowKey : $statePath.'.'.$rowKey.'.'.$columnKey ;
-                            @endphp
-                            <td class="p-2 text-center">
-                                <input
-                                    wire:key="{{ $id }}.{{ $rowKey }}"
+            @foreach($rowData as $rowKey => $rowValue)
+                <tr>
+                    <td class="p-2">{{ $rowValue }}</td>
+                    @foreach($columnData as $columnKey => $columnValue)
+                        @php
+                            $supStatPath = ($pilColor === 'radio') ? $statePath.'.'.$rowKey : $statePath.'.'.$rowKey.'.'.$columnKey ;
+                        @endphp
+                        <td class="p-2 text-center">
+                            <input
+                                    @if($isOptionDisabled($columnKey,'') && $isOptionDisabled($rowKey,'')) disabled @endif
```

Comment:
You can add an OR'd check here for `$isDisabled()` so we can disabled the whole component if needed.

```
@if($isDisabled() || $isOptionDisabled($columnKey,'') && $isOptionDisabled($rowKey,'')) disabled @endif
```